clients/ui/bff/api/helpers.go

```diff
@@ -11,6 +11,8 @@ import (
 
 type Envelope map[string]interface{}
 
+type TypedEnvelope[T any] map[string]T
```

Comment:
@alexcreasy in a next iteration shall we remove the untyped one?

Reply:
@ederign yes, I'd like to, the only reason I didn't just change it was for speed right now :)